python_modules/dagster/dagster/_utils/schedules.py

```diff
@@ -642,14 +668,16 @@ def cron_string_iterator(
             known_schedule_type = ScheduleType.WEEKLY
         elif all(is_numeric[0:2]) and all(is_wildcard[2:]):  # daily
             known_schedule_type = ScheduleType.DAILY
-        elif is_numeric[0] and all(is_wildcard[1:]):  # hourly
+        elif all(cron_part != "*" for cron_part in cron_parts[0]) and all(
+            is_wildcard[1:]
+        ):  # hourly
```

Comment:
any risk of the `all([]) == True` situation causing this to fire inadvertently?

Reply:
hmmm i don't think so in practice, but can't hurt to be explicit just in case